content/pages/contact.md

```diff
@@ -45,7 +45,7 @@ Fund](http://www.student.math.uwaterloo.ca/~mefcom/), and comfortable and clean
 desk space. We also have a library including feminist tech publications and
 various technical resources available for you to read during office hours. 
 The schedule for this term's office hours is 
-[here]({filename}/F2016/F16-Office-Hours.md).
+[here]({filename}/W2017/W17-Office-Hours.md).
```

Comment:
this was already merged hmm, you should merge master into your current branch so this change isn't in this pull request